sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/KafkaBundle.java

```diff
@@ -279,15 +282,18 @@ public <K, V> KafkaConsumer<K, V> createConsumer(
    * @param <V> Value object type
    * @return a new kafka producer
    */
-  public <K, V> KafkaProducer<K, V> createProducer(
+  public <K, V> Producer<K, V> createProducer(
```

Comment:
Sorry, can't mark the right line here: Can you fix the documentation in L271? It says "Creates a new Kafka **Consumer**..."

Reply:
Documentation updated